pkg/interactive/interactive.go

```diff
@@ -106,6 +108,31 @@ func parseInt(str string) (num int, err error) {
 	return strconv.Atoi(str)
 }
 
+// Asks for multiple options selection
+func GetMultipleOptions(input Input) (res []string, err error) {
+	dflt, ok := input.Default.(string)
+	if !ok {
+		dflt = ""
+	}
+	question := input.Question
+	if !input.Required {
```

Comment:
This should be `if !input.Required && dflt == "" {`, otherwise it shouls the prompt with both "(optional)" and the pre-populated value, which has caused confusion in the past.